### PR TITLE
Add SBOM check for enterprise firefly

### PR DESCRIPTION
**Pull request by Chengxuan**
adding and fixing sbom checks

### CODE REVIEW

File: docker_build.sh [hunk 1]
```diff
@@ -16,8 +16,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -e
 echo $@
 
+if [[ ! -x `which jq` ]]; then echo "Please install \"jq\" to continue"; exit 1; fi
```
Comment (matthew1001) [inline docker_build.sh:22]: Looks like this is a dup of the line below?

**Review by matthew1001 — APPROVED** [review]
lgtm, one minor comment

File: docker_build.sh [hunk 2]
```diff
@@ -16,8 +16,11 @@
...
+
```
Comment (Chengxuan) [inline docker_build.sh:23]:
```suggestion
```

File: docker_build.sh (hunk 1 above)
Comment (Chengxuan) [reply docker_build.sh]: thank you. comment addressed

**Pull request merged**